recipes/source.rb

```diff
@@ -43,9 +43,36 @@
   package %w(libssl-dev zlib1g-dev libapr1-dev libcurl4-gnutls-dev)
 end
 
-gem_package 'passenger' do
-  version node['passenger']['version']
-  gem_binary "#{node['passenger']['bin_dir']}/gem" unless node['passenger']['bin_dir'].nil?
+if node['passenger']['enterprise']
+  secrets_databag = chef_vauilt_item(node['passenger']['data_bag_name'], node['passenger']['data_bag_item'])
```

Comment:
I originally had these changes on a separate forked cookbook that I had tested against, but must have incorporated a few typos here and never tested against this branch.

I have fixed the typos and other missing pieces and have functionally tested my forked master branch.